### PR TITLE
Memory leaks fixes

### PR DESCRIPTION
**Pull request by JSibir**
\+ stack.orderFrames() is called only if numberOfFrames > 1

### CODE REVIEW

File: src/controls/controls.trackball.js [hunk 1]
```diff
@@ -637,14 +637,29 @@ import {EventDispatcher, Quaternion, Vector2, Vector3} from 'three';
...
 
-    this.domElement.addEventListener('contextmenu', function(event) {
- event.preventDefault();
-}, false);
+    function contextmenu(event) {
+      event.preventDefault();
+    }
 
-    this.domElement.addEventListener('mousedown', mousedown, false);
+    this.dispose = function() {
```
[the remark this answers is not in the document]
Comment (JSibir) [reply src/controls/controls.trackball.js]: controls.trackballortho has this function

So, it should be added to controls.trackball, or removed from controls.trackballortho?

[the remark this answers is not in the document]
Comment (NicolasRannou) [reply src/controls/controls.trackball.js]: it seems only those have to be disposed: https://github.com/FNNDSC/ami/blob/dev/src/controls/controls.trackball.js#L644-L654

[the remark this answers is not in the document]
Comment (JSibir) [reply src/controls/controls.trackball.js]: You suggest to remove these lines too:
https://github.com/FNNDSC/ami/blob/dev/src/controls/controls.trackballortho.js#L430-L431
Am I correct?

[the remark this answers is not in the document]
Comment (NicolasRannou) [reply src/controls/controls.trackball.js]: yes!

[the remark this answers is not in the document]
Comment (JSibir) [reply src/controls/controls.trackball.js]: Done

**Comment by NicolasRannou** [comment]
Thanks!

**Pull request merged**